src/components/Buttons.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+
+function Buttons({clickHandler, textButton, disableButton, typeButton}) {
+    return(
+            <button
+                className="reset_send"
+                type={typeButton}
+                disabled={disableButton}
+                onClick={clickHandler}
+
+            >{textButton}
+            </button>
+
+    )
+}
+
+
+export default Buttons;
```

Comment:
Dit component is nu een doorgeefluik. Zelf zou ik hier niet een apart component voor maken, maar gewoon direct het html button element gebruiken. Het zorgt nu niet perse voor code reductie namelijk